source/playbooks/NIST80053/ssmdocs/scripts/test/test_parse_event.py

```diff
@@ -0,0 +1,177 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Why is this included? As far as I know, the playbooks now use a common parse input script.